OnlyR.Core/Recorder/AudioRecorder.cs

```diff
@@ -21,6 +21,7 @@ public sealed class AudioRecorder : IDisposable
         private const int VuSpeed = 5;
 
         private LameMP3FileWriter? _mp3Writer;
+        private WaveFileWriter? _waveFileWriter;
```

Comment:
I'd prefer a single instance of Stream rather than a separate one for each codec?

private Stream? _audioWriter;

What do you think?

Reply:
Both writers are made by NAudio, that's why I chose to use WaveFileWriter. I agree, it could be unified by just one Stream, but I'm not sure how to do so.

Reply:
Replace _mp3Writer and _waveWriter with a single _audioWriter of type Stream (the base of both of the NAudio writers). If you're unsure, then please leave it as it is. Thanks!

Reply:
Oh right! I get it now. Sure thing, I'll do it.🙂
Will be back with it soon.